### ISSUE TITLE
Required metadata fields don't display the * symbol

### ISSUE THREAD

**Issue by AgustinAlfieri** [DSpace/DSpace#8770]
I'm using Dspace 7.5 and basing my repository's submission-forms on the type-bind feature. For some types of required metadata fields with certain configurations the `*` symbol is not displayed and if you fail to complete it, the red box and required text won't appear (related to: https://github.com/DSpace/DSpace/issues/8769).

1. Modify the [dspace]/config/submission-form.xml file making the field "type" the first one and adding the "type-bind" tag to the other fields (https://wiki.lyrasis.org/display/DSDOC7x/Submission+User+Interface#SubmissionUserInterface-ItemtypeBasedMetadataCollection).
2.  Add a field with these specifications: **repeatable=true, input-type=list, with required and type-bind tags**.
3. Check if the * symbol appears on those fields.
4. Try to make a submission without completing those fields and see if the red box and required text appear.

**Comment by alexandrevryghem** [comment]
Could you please check if this bug is still present on the latest version of DSpace normally this should have been fixed with PR [2133](https://github.com/DSpace/dspace-angular/pull/2133)

**Comment by AgustinAlfieri** [comment]
Yes, the * symbol appears after I updated to the latest version of Dspace-Angular, Thanks!
However the red box issue (https://github.com/DSpace/DSpace/issues/8769) remains, but since there's already an issue for that, I'm closing this one.
